### PR TITLE
Don't output to stdout

### PR DESCRIPTION
**Pull request by som-snytt**
Some noise in the build output.

```
'13' is not a valid choice for '-target'
bad option: '-target:jvm-13'
'msil' is not a valid choice for '-target'
bad option: '-target:msil'
```

### CODE REVIEW

File: test/junit/scala/tools/nsc/settings/TargetTest.scala [hunk 1]
```diff
@@ -13,23 +13,30 @@
 package scala.tools.nsc
 package settings
 
-import org.junit.{Assert, Test}
+import org.junit.{Assert, Test}, Assert.{assertEquals, assertFalse, assertTrue, fail}
```
Comment (hrhino) [inline test/junit/scala/tools/nsc/settings/TargetTest.scala:16]: What happened to the proposal for

```scala
import org.junit.{Assert.{assertEquals,assertFalse,assertTrue,fail}, Test}
```
?

File: test/junit/scala/tools/nsc/settings/TargetTest.scala [hunk 2]
```diff
@@ -50,8 +57,10 @@ class TargetTest {
     check("-target:jvm-12", "12")
     check("-target:12", "12")
 
-    checkFail("-target:jvm-13") // not yet...
-    checkFail("-target:msil") // really?
+    checkFail("-target:jvm-6")    // no longer
```
Comment (hrhino) [inline test/junit/scala/tools/nsc/settings/TargetTest.scala:60]: Technically, it never was...

File: test/junit/scala/tools/nsc/settings/TargetTest.scala [hunk 3]
```diff
@@ -50,8 +57,10 @@ class TargetTest {
...
+    checkFail("-target:jvm-13")   // not yet...
```
Comment (hrhino) [inline test/junit/scala/tools/nsc/settings/TargetTest.scala:61]: Uncomfortably soon!

**Review by hrhino — APPROVED** [review]
Thanks. I know I was whinging on the other thread about unused imports, but I apparently never run junit tests.

File: test/junit/scala/tools/nsc/settings/TargetTest.scala (hunk 3 above)
Comment (som-snytt) [reply test/junit/scala/tools/nsc/settings/TargetTest.scala:61]: Yeah I decided not to tweak it yet. Seth needs stuff to do later, probably as part of the community build.

File: test/junit/scala/tools/nsc/settings/TargetTest.scala (hunk 2 above)
Comment (som-snytt) [reply test/junit/scala/tools/nsc/settings/TargetTest.scala:60]: My emoji attempt at wistful melancholy.

Comment (hrhino) [reply test/junit/scala/tools/nsc/settings/TargetTest.scala:60]: Valid, but not recommended, as they say.

File: test/junit/scala/tools/nsc/settings/TargetTest.scala (hunk 1 above)
Comment (dwijnand) [reply test/junit/scala/tools/nsc/settings/TargetTest.scala:16]: Hello Rust!

**Pull request merged**